plugins/TagFix_Brand.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Noémie Lehuby 2020                                         ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from plugins.Plugin import TestPluginCommon
+from plugins.Plugin import Plugin
+from modules.downloader import urlread
+import json
+
+
+class TagFix_Brand(Plugin):
+
+    def init(self, logger):
+        Plugin.init(self, logger)
+        self.errors[31301] = self.def_class(item=3130, level=2, tags=['brand', 'fix:chair'],
```

Comment:
```suggestion
        self.errors[31301] = self.def_class(item=3130, level=3, tags=['brand', 'fix:chair'],
```
Since is only improvement and not fixing the data, I think you could move it to level 3?

Reply:
sure